fullstack/packages/server/README.md

```diff
@@ -0,0 +1,73 @@
+<p align="center">
+  <a href="http://nestjs.com/" target="blank"><img src="https://nestjs.com/img/logo_text.svg" width="320" alt="Nest Logo" /></a>
+</p>
+
+[circleci-image]: https://img.shields.io/circleci/build/github/nestjs/nest/master?token=abc123def456
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- 59ace337-4c1a-46e9-babe-a3e66dcd1a24 -->

Reply:
I think all external PR might need to be excluded from cycode secret check, since this repo is for candidates to submit their interview-assignments(which will NOT get merged)

Reply:
Would it make sense to have a dedicated github org for china public or interview repos, so we can apply a differen security policy?

Reply:
for this case, we can leave it alone as it is a demo purpose (not a real token)

and we just keep this security policy, where it should also be part of the consideration that if such code should be commit or not